src/dotnet/Azure.SDK.ChangelogGen/Azure.SDK.ChangelogGen/ChangeLogResult.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.SDK.ChangelogGen.Compare;
+using Azure.SDK.ChangelogGen.Report;
+
+namespace Azure.SDK.ChangelogGen
+{
+    public class ChangeLogResult
+    {
+        public ChangeSet? ApiChange { get; set; } = null;
+        public StringValueChange? SpecVersionChange { get; set; } = null;
+        public StringValueChange? AzureCoreVersionChange { get; set; } = null;
+        public StringValueChange? AzureResourceManagerVersionChange { get; set; } = null;
+
+        public Release GenerateReleaseNote()
+        {
+            Release report = new Release("release_version", "release_date");
+
+            ReleaseNoteGroup breakingGroup = new ReleaseNoteGroup("Breaking Changes in API");
```

Comment:
These groups do not conform to the required sections in the Azure SDK release notes policy.

Reply:
@ArthurMa1978, could you help to confirm the correct sections to use?

Reply:
Please read the release notes policy that I've linked above.  That defines the sections that all change logs are expected to have.

Reply:
This should be `Breaking Changes`

Reply:
updated. thx